Campaigns/Abusing settingcontent-ms.txt

```diff
@@ -0,0 +1,8 @@
+// Sample query that search for .settingcontent-ms that has been downloaded from the web
+// through Microsoft Edge, Internet Explorer, Google Chrome and Mozilla Firefox
+// Require Windows 10 Build 17686+ 
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com
+FileCreationEvents
+| where InitiatingProcessFileName in~ ("browser_broker.exe", "chrome.exe", "iexplore.exe", "firefox.exe")
```

Comment:
maybe add outlook.exe as well - is it relevant?

Reply:
Added